microsoftgraph/microsoft-graph-explorer-v4
# Fix: Try it signing out experience

## Overview
Fixes #1579 

### Demo

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions
Check out this branch and start the application.
On the url, add a 'theme' query parameter like so: http://localhost:3000?theme=theme
Sign in to the app using any account
Click on your profile information. Now click on Sign out. 
Notice the fix
